tests/unit/config/test_phases.py

```
@@ -277,3 +278,80 @@ def test_request_rate_adaptive_control_rejects_rate_series(self) -> None:
+    @pytest.mark.parametrize(
+        "median",
+        [
+            param(0.5, id="median_equals_mean"),
+            param(1.5, id="median_above_mean"),
+        ],
+    )  # fmt: skip
+    def test_median_not_below_mean_rejected_with_computed_mean(
+        self, median: float
+    ) -> None:
+        # mean gap = users / rate = 5 / 10.0 = 0.5s.
+        with pytest.raises(ValueError, match="0.5"):
```

Ruff (0.15.21)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)